vip_spec.xsd

```diff
@@ -8,6 +8,7 @@ reference elements for polling location
   <xs:element name="VipObject">
     <xs:complexType>
       <xs:choice maxOccurs="unbounded">
+
```

Comment:
Are the random empty lines being added a style thing?  We should probably document that if so.